pnpm-lock.yaml

```diff
@@ -1442,7 +1444,7 @@ packages:
     engines: {node: ^12.13.0 || ^14.15.0 || ^16.10.0 || >=17.0.0}
     dependencies:
       '@jest/types': 28.1.3
-      '@types/node': 18.6.3
+      '@types/node': 17.0.45
```

Comment:
huh?

Reply:
This is coming from a `git merge main`. Thanks for spotting it, I'll revert to `@types/node@18.6.3` (even though we should depend on `@types/node@14.17.0` imho, as that's the oldest Node.js version we support)